node-api/src/events.rs

```diff
@@ -180,7 +180,7 @@ impl EventDetails {
 
 		// topics come after the event data in EventRecord. They aren't used for
 		// anything at the moment, so just decode and throw them away.
-		let _topics = Vec::<Hash>::decode(input)?;
+		let _topics = Vec::<H256>::decode(input)?;
```

Comment:
Hardcoded H256 for now.. as they're not used and are unlikely to be different from H256 at the moment.